bulma/layout/section.sass

```diff
@@ -1,5 +1,5 @@
 .section
-  background: white
+  background: $white
```

Comment:
@billoran8  I tried providing a value for $white and sections are unaffected.  Is this maybe not specific enough? Should it be section.section?